util/topsql/tracecpu/profile.go

```diff
@@ -24,31 +24,49 @@ import (
 	"runtime/pprof"
 	"strconv"
 	"sync"
-	"sync/atomic"
 	"time"
 
 	"github.com/google/pprof/profile"
-	"github.com/pingcap/tidb/sessionctx/variable"
+	"github.com/pingcap/tidb/metrics"
 	"github.com/pingcap/tidb/util"
 	"github.com/pingcap/tidb/util/hack"
 	"github.com/pingcap/tidb/util/logutil"
+	"go.uber.org/atomic"
 	"go.uber.org/zap"
 )
 
 const (
 	labelSQL        = "sql"
 	labelSQLDigest  = "sql_digest"
 	labelPlanDigest = "plan_digest"
+
+	// DefPrecisionSeconds indicates that the default PrecisionSeconds is 10000
+	DefPrecisionSeconds = 1
+
+	// DefEnablePProfSQLCPU indicates that the default EnablePProfSQLCPU is false
+	DefEnablePProfSQLCPU = 0
 )
 
-// GlobalSQLCPUProfiler is the global SQL stats profiler.
-var GlobalSQLCPUProfiler = newSQLCPUProfiler()
+var (
+	// GlobalSQLCPUProfiler is the global SQL stats profiler.
+	GlobalSQLCPUProfiler = newSQLCPUProfiler()
+
+	// EnablePProfSQLCPU is true means want to keep the `sql` label. Otherwise, remove the `sql` label.
+	EnablePProfSQLCPU = atomic.NewBool(false)
+
+	// PrecisionSeconds indicates profile interval.
+	PrecisionSeconds = atomic.NewInt64(DefPrecisionSeconds)
```

Comment:
They should be a config of sqlCPUProfiler I think, from this module's perspective.